sdk_lib/enter_chroot.sh

```diff
@@ -185,9 +185,9 @@ generate_locales() {
     # with long multibyte strings.  Newer setups have this fixed,
     # but locale-gen doesn't need to be run in any locale in the
     # first place, so just go with C to keep it fast.
-    PATH="/usr/sbin:/usr/bin:/sbin:/bin" LC_ALL=C \
-      chroot "${FLAGS_chroot}" locale-gen -q -u \
-      -G "$(printf '%s\n' "${gen_locales[@]}")"
+    chroot "${FLAGS_chroot}" /usr/bin/env \
```

Comment:
The path that the chroot needs is very different than the path of my host. Before, we could override the host path and use that in the chroot because it was compatible with our systems. This change just ensures that the host and chroot paths stay separate (which seems more correct).